lpcompiler.pas

```diff
@@ -3131,6 +3156,7 @@ constructor TLapeCompiler.Create(
   FDefines.Duplicates := dupIgnore;
   FDefines.CaseSensitive := LapeCaseSensitive;
   FConditionalStack := TLapeConditionalStack.Create(0);
+  FHasHintDefine := False;
```

Comment:
Defined in `TLapeCompilerBase`, so it should be initialized there as well? (if it's needed)